### ISSUE TITLE
Admin: Keyboard focus is not visible for the bypass blocks

### ISSUE THREAD

**Issue by XiuJuanXia** [mmiron-HC/WST-RSAMS#150]
URL: https://rsams.dev.cloud.openplus.ca/en/user/197
User Agent: Chrome (v100), Firefox (v98)
Language: Both

Issue Description: When tabbing through, above the "Home", there are three bypass blocks links (Skip to main content, Skip to "About government" and Switch to basic HTML version). Keyboard focus indicator is not visible for these three bypass blocks links.
`<nav aria-label="Skip links">
      <ul id="wb-tphp" class="wb-init wb-disable-inited">
        <li class="wb-slc">
          <a class="wb-sl" href="#wb-cont">
            Skip to main content
          </a>
        </li>
        <li class="wb-slc">
          <a class="wb-sl" href="#wb-info">
            Skip to "About government"
          </a>
        </li>
      <li class="wb-slc"><a class="wb-sl" href="?wbdisable=true" rel="alternate">Switch to basic HTML version</a></li></ul>
    </nav>`
Steps to reproduce:

1. Login as an admin
2. Tabbing through any page
3. Three tabbing stop before focus is placed on the "Home"
4. Inspect the source code and there are three bypass blocks links that don't have keyboard focus indicator

Expected results: When tabbing to the three bypass blocks links, the three links should be visible when focus is on.
